physlr/physlr.py

```diff
@@ -1201,6 +1207,27 @@ def physlr_overlap(self):
             (u, v) for bxs in progress(mxtobxs.values()) for u, v in itertools.combinations(bxs, 2))
         print(int(timeit.default_timer() - t0), "Loaded", len(edges), "edges", file=sys.stderr)
 
+        edge_list = list(edges)
+        neighbour_min = defaultdict(list)
+
+        for edge in edge_list:
+            neighbour_min[edge[0]] += bxtomxs[edge[1]]
+            neighbour_min[edge[1]] += bxtomxs[edge[0]]
```

Comment:
What about doing the recalculation of the edge weight after adding the edges to the graph (line 1231)? Then, you could just something like `bxtomxs[v] for v in g.neighbours(bx)` to find the minimizers of the neighbours of a given node

Reply:
well I can benchmark that later and see the difference in time